### PR TITLE
feat: add DAOLama project

### PR DESCRIPTION
**Pull request by mp0wer**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### methodology (what is being counted as tvl, how is tvl being calculated): 

Counts the pool size as the TVL. Borrowed coins are not counted towards the TVL.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/daolama exports TVL: 
        
 
 
    ton-borrowed              104.88 k
    ton                       175.03 k
    borrowed                  104.88 k
    
    total                    175.03 k

**Comment by g1nt0ki** [comment]
@mp0wer thanks for the PR, unfortunately we currently do not support NFTs in tvl for chains other than ethereum, even in ethereum only a handful with oracle support. Reason why we are doing this: https://github.com/DefiLlama/DefiLlama-Adapters/pull/5002#issuecomment-1377724981

We can count the TON in pool not yet lent out as tvl and borrowed amount under borrowed field (your current implementation for this need not change), is it possible to get only the remaining TON liquidity in a pool via api call?

**Comment by pz6tnk** [comment]
@g1nt0ki  thanks for the answer! Is it possible to count value of NFTs as borrowed amount? We estimate NFTs and issue our real TON coins so we do not interested to issue loans for low-liquidity or overestimated NFTs. 

We can mention that in the description. We "vote" for the value of NFTs by our own funds so we think it's fair to count minimum LTV as borrowed amount.

**Comment by g1nt0ki** [comment]
@mp0wer we give users an option to include borrowed amount in  total tvl
image


We dont like to include borrowed in tvl by default as some protocols support leveraging/looping. 

You want to count the nfts rented out as borrowed? Bit of grey area, yes you can, can you add what you are including in tvl and borrowed in the methodology field of the adapter file? if users/other protocols complain about this, we might have revisit/change this.

**Comment by llamatester** [comment]
The adapter at projects/daolama exports TVL: 
        
 
 
    ton-borrowed              101.82 k
    ton                       107.92 k
    borrowed                  101.82 k
    
    total                    107.92 k

**Comment by llamatester** [comment]
The adapter at projects/daolama exports TVL: 
        
 
 
    ton                       8.20 k
    ton-borrowed              103.52 k
    borrowed                  103.52 k
    
    total                    8.20 k

File: projects/daolama/tvl.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const { get } = require("../helper/http");
+const { transformBalances } = require("../helper/portedTokens");
+const ADDRESSES = require("../helper/coreAssets.json");
+const { BASE_API_URL, POOL_ADDRESS } = require("./constants");
+const nullAddress = ADDRESSES.null;
+
+async function tvl(chain, timestamp, chainBlocks, { api }) {
+  const balances = {};
+  const result = await get(`${BASE_API_URL}/api/v1/analytics/borrowed`);
+  sdk.util.sumSingleBalance(balances, nullAddress, result.value, api.chain);
```
[the remark this answers is not in the document]
Comment (mp0wer) [reply projects/daolama/tvl.js]: Removed

**Pull request merged**